hicexplorer/hicBuildMatrix.py

```diff
@@ -490,139 +580,41 @@ def enlarge_bins(bin_intervals, chrom_sizes):
     return bin_intervals
 
 
-def main(args=None):
-    """
-    Reads line by line two bam files that are not sorted.
-    Each line in the two bam files should correspond
-    to the mapped position of the two ends of a Hi-C
-    fragment.
-
-    Each mate pair is assessed to determine if it is
-    a valid Hi-C pair, in such case a matrix
-    reporting the counts of mates is constructed.
-
-    A bam file containing the valid Hi-C reads
-    is also constructed
-    """
-
-    # check pysam version
-    from distutils.version import LooseVersion
-    if LooseVersion(pysam.__version__) < LooseVersion("0.8.3"):
-        exit("\n*ERROR*\n\nVersion of pysam has to be higher than 0.8.3. Current installed version is {}\n".format(pysam.__version__))
-
-    args = parse_arguments().parse_args(args)
-
-    # check that the log folder is valid
-    try:
-        QC.make_sure_path_exists(args.QCfolder)
```

Comment:
You are right. I think it was getting lost when I did the manual merge with upstream.